### DISCUSSION TITLE
Hebrew Layout

### DISCUSSION THREAD

**Discussion by giluriel** [8VIM/8VIM#143] [Language Layout]
![8pen-hebrew-layout-with codes](https://user-images.githubusercontent.com/6133929/102225220-9e5e0600-3eef-11eb-8c35-20f0441aebf7.jpg)

Hi,
This is the Hebrew layout.
In red are the ascii codes 
TNX

**Comment by flide** [comment]
For everyone reading. Check #75 for more information on this discussion and then continue here.

**Comment by giluriel** [comment]
Hi, Any news with that?

Just to remind that people who use more than one language cannot use the 8vim yet... it is not practical to switch keyboards each time  language switch is needed.
We are waiting ...  pls .... :) :)

**Reply by adaschma** [reply]
Hi @giluriel 
I'll create a hebrew layout for you. Could you type the layout as the 8 groups of 4 chars here in counter clock wise order starting with  א  ? Thanks.

**Reply by adaschma** [reply]
Hi @giluriel 

It's been almost a week and I haven't heard anything from you. I just wanted to check in to make sure that it didn't get forgotten. If what I wrote was unclear please don't hesitate to ask. Hope to hear from you soon :-)

**Reply by giluriel** [reply]
> Hi @giluriel
> I'll create a hebrew layout for you. Could you type the layout as the 8 groups of 4 chars here in counter clock wise order starting with א ? Thanks.

Hi Adam,
Sorry for the delay, Somehow I missed your reply.

The image in the beginning of this post is the correct order of the hebrew letters in 8 pen. (in red are their matching ascii codes).

Do you need it in another way? format?
(attaching the image again for your convenience)
![102225051-6ce53a80-3eef-11eb-84c9-72646e34deb2](https://user-images.githubusercontent.com/6133929/120100717-d8902c80-c14a-11eb-949c-ca5f1c6b071c.jpg)

**Reply by adaschma** [reply]
Hi @giluriel,

No worries. Glad I asked again :-).

I'm afraid the ascii codes don't help. Is it possible to type them here in the order I suggest? Alternative I could work with the unicode codes (though working with the text directly is easier).

**Comment by flide** [comment]
this is now done
